### PR TITLE
chore(react-tabster): upgrade tabster to v4.4.2

### PR DESCRIPTION
**Pull request by bsunderhus**
## New Behavior

1. Bumps tabster version to 4.4.0

## Related Issue(s)

- Fixes https://github.com/microsoft/fluentui/issues/27487
- Fixes https://github.com/microsoft/fluentui/issues/26824

### CODE REVIEW

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27540/perf-test-react-components/Avatar-mount_base.html">690</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27540/perf-test-react-components/Avatar-mount.html">717</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27540/perf-test-react-components/Button-mount_base.html">390</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27540/perf-test-react-components/Button-mount.html">386</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27540/perf-test-react-components/Field-mount_base.html">1201</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27540/perf-test-react-components/Field-mount.html">1200</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27540/perf-test-react-components/FluentProvider-mount_base.html">751</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27540/perf-test-react-components/FluentProvider-mount.html">790</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27540/perf-test-react-components/FluentProviderWithTheme-mount_base.html">98</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27540/perf-test-react-components/FluentProviderWithTheme-mount.html">106</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27540/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">94</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27540/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">87</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27540/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">105</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27540/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">93</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27540/perf-test-react-components/InfoButton-mount_base.html">16</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27540/perf-test-react-components/InfoButton-mount.html">16</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27540/perf-test-react-components/MakeStyles-mount_base.html">1154</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27540/perf-test-react-components/MakeStyles-mount.html">1158</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27540/perf-test-react-components/Persona-mount_base.html">1896</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27540/perf-test-react-components/Persona-mount.html">1841</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27540/perf-test-react-components/SpinButton-mount_base.html">1501</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27540/perf-test-react-components/SpinButton-mount.html">1486</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: change/@fluentui-react-tabster-15456040-19ed-4576-b0d5-918ea610f2dd.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-tabster-15456040-19ed-4576-b0d5-918ea610f2dd.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27540&commitId=2f5a0393532776d3d6a6e489c5c34d129f543886&env=prod)

✅ There was 0 screenshots added, 2 screenshots removed, 1045 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports                                                                                                                                                             |      Baseline (minified/GZIP) |                            PR |                                                                                                                                                                                                      Change |
| :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: | ----------------------------: | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
| <samp>react-accordion</samp> <br /> <abbr title='bundle-size/Accordion.fixture.js'>Accordion (including children components)</abbr>                                           |  `88.959 kB`<br />`26.571 kB` |   `91.011 kB`<br />`27.14 kB` | `2.052 kB` <br />`569 B`  |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.All.fixture.js'>Card - All</abbr>                                                                                |  `83.566 kB`<br />`23.927 kB` |  `85.849 kB`<br />`24.554 kB` | `2.283 kB` <br />`627 B`  |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.fixture.js'>Card</abbr>                                                                                          |   `78.385 kB`<br />`22.46 kB` |  `80.667 kB`<br />`23.105 kB` | `2.282 kB` <br />`645 B`  |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `207.606 kB`<br />`57.581 kB` | `209.379 kB`<br />`58.187 kB` | `1.773 kB` <br />`606 B`  |
| <samp>react-datepicker-compat</samp> <br /> <abbr title='bundle-size/DatePicker.fixture.js'>DatePicker Compat</abbr>                                                          | `251.351 kB`<br />`67.358 kB` | `253.124 kB`<br />`67.924 kB` | `1.773 kB` <br />`566 B`  |
| <samp>react-dialog</samp> <br /> <abbr title='bundle-size/Dialog.fixture.js'>Dialog (including children components)</abbr>                                                    |  `93.319 kB`<br />`27.389 kB` |  `94.923 kB`<br />`27.903 kB` | `1.604 kB` <br />`514 B`  |
| <samp>react-infobutton</samp> <br /> <abbr title='bundle-size/InfoButton.fixture.js'>InfoButton</abbr>                                                                        | `130.671 kB`<br />`39.464 kB` | `132.276 kB`<br />`40.036 kB` | `1.605 kB` <br />`572 B`  |
| <samp>react-infobutton</samp> <br /> <abbr title='bundle-size/InfoLabel.fixture.js'>InfoLabel</abbr>                                                                          |  `133.97 kB`<br />`40.505 kB` | `135.572 kB`<br />`41.045 kB` | `1.602 kB` <br />`540 B`  |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.fixture.js'>Menu (including children components)</abbr>                                                          | `131.132 kB`<br />`39.675 kB` |  `133.186 kB`<br />`40.25 kB` | `2.054 kB` <br />`575 B`  |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.Selectable.fixture.js'>Menu (including selectable components)</abbr>                                             | `134.268 kB`<br />`40.211 kB` | `136.322 kB`<br />`40.782 kB` | `2.054 kB` <br />`571 B`  |
| <samp>react-popover</samp> <br /> <abbr title='bundle-size/Popover.fixture.js'>Popover</abbr>                                                                                 | `117.556 kB`<br />`35.812 kB` | `119.158 kB`<br />`36.354 kB` | `1.602 kB` <br />`542 B`  |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/DataGrid.fixture.js'>DataGrid</abbr>                                                                                 | `150.983 kB`<br />`41.093 kB` | `153.037 kB`<br />`41.714 kB` | `2.054 kB` <br />`621 B`  |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableAsDataGrid.fixture.js'>Table as DataGrid</abbr>                                                                 | `139.125 kB`<br />`35.768 kB` | `141.178 kB`<br />`36.325 kB` | `2.053 kB` <br />`557 B`  |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                |         Size (minified/GZIP) |
| ---------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------: |
| <samp>global-context</samp> <br /> <abbr title='bundle-size/createContext.fixture.js'>createContext</abbr>                                                       |         `533 B`<br />`341 B` |
| <samp>global-context</samp> <br /> <abbr title='bundle-size/createContextSelector.fixture.js'>createContextSelector</abbr>                                       |         `560 B`<br />`352 B` |
| <samp>priority-overflow</samp> <br /> <abbr title='bundle-size/overflow-manager.fixture.js'>createOverflowManager</abbr>                                         |   `3.194 kB`<br />`1.314 kB` |
| <samp>react-alert</samp> <br /> <abbr title='bundle-size/Alert.fixture.js'>Alert</abbr>                                                                          | `96.725 kB`<br />`23.155 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/Avatar.fixture.js'>Avatar</abbr>                                                                       |  `60.39 kB`<br />`15.605 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroup.fixture.js'>AvatarGroup</abbr>                                                             |   `18.137 kB`<br />`6.67 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroupItem.fixture.js'>AvatarGroupItem</abbr>                                                     |  `76.651 kB`<br />`20.11 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/Badge.fixture.js'>Badge</abbr>                                                                          |  `26.063 kB`<br />`7.621 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/CounterBadge.fixture.js'>CounterBadge</abbr>                                                            |   `27.046 kB`<br />`7.93 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/PresenseBadge.fixture.js'>PresenceBadge</abbr>                                                          |  `34.535 kB`<br />`8.716 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/Button.fixture.js'>Button</abbr>                                                                       |  `39.827 kB`<br />`9.987 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/CompoundButton.fixture.js'>CompoundButton</abbr>                                                       | `46.999 kB`<br />`11.456 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/MenuButton.fixture.js'>MenuButton</abbr>                                                               | `44.514 kB`<br />`11.332 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/SplitButton.fixture.js'>SplitButton</abbr>                                                             | `53.009 kB`<br />`12.898 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/ToggleButton.fixture.js'>ToggleButton</abbr>                                                           | `58.199 kB`<br />`11.907 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardFooter.fixture.js'>CardFooter</abbr>                                                                 |    `9.11 kB`<br />`3.824 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardHeader.fixture.js'>CardHeader</abbr>                                                                 |  `11.036 kB`<br />`4.528 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardPreview.fixture.js'>CardPreview</abbr>                                                               |   `9.915 kB`<br />`4.173 kB` |
| <samp>react-checkbox</samp> <br /> <abbr title='bundle-size/Checkbox.fixture.js'>Checkbox</abbr>                                                                 | `37.097 kB`<br />`11.277 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Combobox.fixture.js'>Combobox (including child components)</abbr>                                    | `89.865 kB`<br />`28.545 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Dropdown.fixture.js'>Dropdown (including child components)</abbr>                                    | `88.549 kB`<br />`28.267 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr> | `68.075 kB`<br />`18.416 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>               | `38.778 kB`<br />`12.337 kB` |
| <samp>react-divider</samp> <br /> <abbr title='bundle-size/Divider.fixture.js'>Divider</abbr>                                                                    |   `19.94 kB`<br />`6.725 kB` |
| <samp>react-field</samp> <br /> <abbr title='bundle-size/Field.fixture.js'>Field</abbr>                                                                          |  `21.571 kB`<br />`7.437 kB` |
| <samp>react-image</samp> <br /> <abbr title='bundle-size/Image.fixture.js'>Image</abbr>                                                                          |   `13.97 kB`<br />`4.978 kB` |
| <samp>react-input</samp> <br /> <abbr title='bundle-size/Input.fixture.js'>Input</abbr>                                                                          |  `26.784 kB`<br />`8.141 kB` |
| <samp>react-label</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Label</abbr>                                                                          |  `12.645 kB`<br />`4.609 kB` |
| <samp>react-link</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Link</abbr>                                                                            |   `12.26 kB`<br />`5.053 kB` |
| <samp>react-overflow</samp> <br /> <abbr title='bundle-size/hooks.fixture.js'>hooks only</abbr>                                                                  |  `11.214 kB`<br />`4.271 kB` |
| <samp>react-persona</samp> <br /> <abbr title='bundle-size/Persona.fixture.js'>Persona</abbr>                                                                    | `67.416 kB`<br />`17.552 kB` |
| <samp>react-portal</samp> <br /> <abbr title='bundle-size/Portal.fixture.js'>Portal</abbr>                                                                       |  `11.649 kB`<br />`4.263 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                            |   `6.446 kB`<br />`2.185 kB` |
| <samp>react-positioning</samp> <br /> <abbr title='bundle-size/usePopper.fixture.js'>usePositioning</abbr>                                                       |  `24.008 kB`<br />`8.798 kB` |
| <samp>react-progress</samp> <br /> <abbr title='bundle-size/ProgressBar.fixture.js'>ProgressBar</abbr>                                                           |  `16.397 kB`<br />`5.846 kB` |
| <samp>react-provider</samp> <br /> <abbr title='bundle-size/FluentProvider.fixture.js'>FluentProvider</abbr>                                                     |  `20.725 kB`<br />`7.104 kB` |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/Radio.fixture.js'>Radio</abbr>                                                                          | `35.635 kB`<br />`11.179 kB` |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/RadioGroup.fixture.js'>RadioGroup</abbr>                                                                |  `18.894 kB`<br />`6.926 kB` |
| <samp>react-select</samp> <br /> <abbr title='bundle-size/Select.fixture.js'>Select</abbr>                                                                       |  `27.992 kB`<br />`9.215 kB` |
| <samp>react-slider</samp> <br /> <abbr title='bundle-size/Slider.fixture.js'>Slider</abbr>                                                                       | `36.922 kB`<br />`11.506 kB` |
| <samp>react-spinbutton</samp> <br /> <abbr title='bundle-size/SpinButton.fixture.js'>SpinButton</abbr>                                                           | `36.693 kB`<br />`10.784 kB` |
| <samp>react-spinner</samp> <br /> <abbr title='bundle-size/Spinner.fixture.js'>Spinner</abbr>                                                                    |  `23.503 kB`<br />`7.217 kB` |
| <samp>react-switch</samp> <br /> <abbr title='bundle-size/Switch.fixture.js'>Switch</abbr>                                                                       |  `32.515 kB`<br />`9.726 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/PrimitivesOnly.fixture.js'>Table (Primitives only)</abbr>                                               | `47.196 kB`<br />`12.732 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSelectionOnly.fixture.js'>Table (Selection only)</abbr>                                            | `86.954 kB`<br />`21.779 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSortOnly.fixture.js'>Table (Sort only)</abbr>                                                      | `86.284 kB`<br />`21.589 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Default.fixture.js'>Text - Default</abbr>                                                           |  `14.977 kB`<br />`5.308 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Wrappers.fixture.js'>Text - Wrappers</abbr>                                                         |  `15.531 kB`<br />`5.215 kB` |
| <samp>react-textarea</samp> <br /> <abbr title='bundle-size/Textarea.fixture.js'>Textarea</abbr>                                                                 |  `30.185 kB`<br />`9.502 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/SingleThemeToken.fixture.js'>Single theme token import</abbr>                                           |           `69 B`<br />`89 B` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsAllThemes.fixture.js'>Teams: all themes</abbr>                                                     |   `31.33 kB`<br />`6.764 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsLightTheme.fixture.js'>Teams: Light theme</abbr>                                                   |  `17.895 kB`<br />`5.162 kB` |
| <samp>react-tooltip</samp> <br /> <abbr title='bundle-size/Tooltip.fixture.js'>Tooltip</abbr>                                                                    | `49.465 kB`<br />`16.844 kB` |
| <samp>react-utilities</samp> <br /> <abbr title='bundle-size/SSRProvider.fixture.js'>SSRProvider</abbr>                                                          |         `180 B`<br />`159 B` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/bbdfcb9795bca865f8b632f30ee5c7156e092fc5'>bbdfcb9795bca865f8b632f30ee5c7156e092fc5</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 FluentUI-v0 No visual regressions between this PR and main
<div id="vrtCommentFluentUI-v0"/>

**Comment by layershifter** [comment]
image

Is it real increase? 💥

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

**Comment by bsunderhus** [comment]
> image
> 
> Is it real increase? 💥

yes, it is. modalizer has more scenarios covered

File: change/@fluentui-react-tabster-15456040-19ed-4576-b0d5-918ea610f2dd.json (hunk 1 above)
Comment (fabricteam) [inline change/@fluentui-react-tabster-15456040-19ed-4576-b0d5-918ea610f2dd.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27540&commitId=27052782478565d67f4c0bec4bd62225d61c55fd&env=prod)

✅ There was 0 screenshots added, 2 screenshots removed, 1045 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>

**Review by ling1726 — APPROVED** [review]

**Pull request merged**